requirements/specifications/device/device-information.md

```diff
@@ -0,0 +1,193 @@
+# Device Information
+
+Document Status: Working Draft
+
+See [Firebolt Requirements Governance](../../governance.md) for more info.
+
+| Contributor     | Organization   |
+| --------------- | -------------- |
+| Andrew Bennett  | Sky            |
+| Piotr Kobzda    | Liberty        |
+| Jeremy LaCivita | Comcast        |
+| Farhan Mahmood  | Liberty        |
+| Jan Pedersen    | Sky            |
+
+## 1. Overview
+This document outlines the requirements for ontological, intrinsic, and product-related device indentification APIs.
+
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**NOT RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in [BCP 14](https://www.rfc-editor.org/rfc/rfc2119.txt) [RFC2119] [RFC8174] when, and only when, they appear in all capitals, as shown here.
+
+## 2. Table of Contents
+- [1. Overview](#1-overview)
+- [2. Table of Contents](#2-table-of-contents)
+- [3. Device Identifiers](#3-device-identifiers)
+  - [3.1. Device Identifier](#31-device-identifier)
+  - [3.2. Anonymized Device Identifier](#32-anonymized-device-identifier)
+  - [3.3. Device Name](#33-device-name)
+- [4. Versions](#4-versions)
+  - [4.1. Platform Version](#41-platform-version)
+  - [4.2. Firmware Version](#42-firmware-version)
+  - [4.3. Debugging Info](#43-debugging-info)
+- [5. Device Classification](#5-device-classification)
+  - [5.1. Type](#51-type)
+  - [5.2. Class](#52-class)
+  - [5.3. Make](#53-make)
+  - [5.4. Model](#54-model)
+  - [5.5. Serial Number](#55-serial-number)
+- [6. Device Proposition](#6-device-proposition)
+  - [6.1. Distributor](#61-distributor)
+  - [6.2. Partner](#62-partner)
+  - [6.3. Propositions](#63-propositions)
+- [7. Cuts](#7-cuts)
+- [8. Refactored into to other modules](#8-refactored-into-to-other-modules)
+  - [8.1. MediaInfo](#81-mediainfo)
+  - [8.2. HDMI](#82-hdmi)
+  - [8.3. Network](#83-network)
+  - [8.4. Analog Output](#84-analog-output)
+  - [8.5. Manage SDK (TBD)](#85-manage-sdk-tbd)
+
+## 3. Device Identifiers
+
+### 3.1. Device Identifier
+The Device module **MUST** expose a read-only, clear-text unique identifier of the device via the `Device.id` method.
```

Comment:
What is the durability of this attribute?  Does it survive device reset?  Is it available even on a new device?